### ISSUE TITLE
Subscript and superscript in LaTeX fragments

### ISSUE THREAD

**Issue by szclsya** [niklasfasching/go-org#12]
When a subscript or a superscript appears in a LaTeX fragment, it should remain untouched so that MathJax or KaTeX can process them. But currently, if a subscript or a superscript presents in a LaTeX fragment, it will be rendered with a `<sub>` or a `<sup>` tag, making it impossible for MathJax to process.

Example code: 
```org
Example code: \(\sum_{i=1}^n a_n\)
```

Output
```html
<p>
Example code: \(∑<sub>i=1</sub>^n a_n\)
</p>
```

**Comment by niklasfasching** [comment]
I pretty much never use latex fragments so support for latex fragments is missing. Should be doable but not sure when I'll get around to working on this again - always open to a PR :)

**Comment by niklasfasching** [comment]
Implemented in [76b157b8](https://github.com/niklasfasching/go-org/commit/76b157b8ced6eafc30abb373667fb62d6520dc83).

See [raw html]https://github.com/niklasfasching/go-org/blob/master/org/testdata/latex.html) and [rendered html](https://niklasfasching.github.io/go-org/#latex.org).

**Comment by szclsya** [comment]
Awesome! Thank you very much for the work on this project.
